app-frontend/src/app/components/modelSearch/modelSearch.html

```diff
@@ -0,0 +1,30 @@
+<div class="navbar secondary-navbar">
+  <div class="navbar-left">
+    <nav ng-show="$ctrl.$state.$current.name === 'market.model'">
+      <a href ui-sref="market.search" class="active"> < Back to search results</a>
+    </nav>
+    <nav ng-show="$ctrl.$state.$current.name === 'market.search'">
+      <ng-submit ng-submit="$ctrl.onSearchAction($ctrl.searchText)">
+        <div class="form-group all-in-one input-dark">
+          <label for="search-input">
+            <i class="icon-search"></i>
+          </label>
+          <input type="text"
+                 id="search-input"
+                 class="form-control input-dark"
+                 placeholder="Search for models"
+                 ng-model="$ctrl.searchText"
+                 ng-submit="$ctrl.onSearchAction($ctrl.searchText)"/>
```

Comment:
Yea, it doesn't do anything right now. spent about 15 mins trying to figure out why then moved on since it's just a skeleton.  Should I remove it in that case?